esp/src/src/nls/bs/hpcc.ts

```diff
@@ -157,6 +164,7 @@
     DaliIP: "Dali IP adresa",
     Dashboard: "Kontrolna ploča",
     DataPatterns: "Uzorci Podataka",
+    DataPatternsDefnNotFound: "",
```

Comment:
Missed that one - pls pick it up from an update I sent

Reply:
added.